Ginger/Ginger/ApplicationModelsLib/POMModels/PomElementsPage.xaml.cs

```diff
@@ -124,6 +141,37 @@ public PomElementsPage(ApplicationPOMModel pom, PomAllElementsPage.eElementsCont
             {
                 DisableDetailsExpander();
             }
+
+            
+        }
+
+        public void DoEndOfRelearnElementsSorting()
```

Comment:
DoEndOfRelearnElementsSorting() should be done in the Compare Wizard and not here in page